2022-09-september/README.md

```diff
@@ -0,0 +1,114 @@
+# 🎒 🍃 ✨ What the hack happened this month at Hack Club?
+
+Hey friends!
+
+<img src="https://ca.slack-edge.com/T0266FRGM-USNPNJXNX-1194588c3884-48" width="16"> [Sam](https://github.com/sampoder), Hack Clubber from Singapore, here. It's been a busy couple of weeks in the Hack Club community, I wanted to email all of you with a rundown of what your fellow Hack Clubbers have been up to this past month! 
```

Comment:
a grammar note: there's lots of places that i think should be semicolons instead of commas; it still makes sense with commas, but unsure if you want to change them?
```suggestion
<img src="https://ca.slack-edge.com/T0266FRGM-USNPNJXNX-1194588c3884-48" width="16"> [Sam](https://github.com/sampoder), Hack Clubber from Singapore, here. It's been a busy couple of weeks in the Hack Club community; I wanted to email all of you with a rundown of what your fellow Hack Clubbers have been up to this past month! 
```

Reply:
~~I'm always scared to use semicolons~~